docs/topics/forms/modelforms.txt

```diff
@@ -1230,10 +1230,42 @@ of a model. Here's how you can do that::
 
 Notice how we pass ``instance`` in both the ``POST`` and ``GET`` cases.
 
+Inline Formsets and ``prefix``
+------------------------------
+
+Inline Formsets are automatically assigned a default ``prefix``. For example, consider the following models and formset::
+
+    from django.db import models
+    from django.forms import inlineformset_factory
+
+    class Author(models.Model):
+        name = models.CharField(max_length=100)
+
+    class Book(models.Model):
+        author = models.ForeignKey(Author, on_delete=models.CASCADE)
+        title = models.CharField(max_length=100)
+
+    BookFormSet = inlineformset_factory(Author, Book, fields=('title',), extra=1)
+
+``BookFormSet`` above will have its ``prefix`` attribute set to 'book_set'. If Book's ForeignKey to Author had a ``related_name``, that would be used instead::
+
+    >>> author = Author.objects.get(name='William Faulkner')
```

Comment:
I was thinking an example would instead go in the formset docs because this same concern is present there. (The default prefix is 'form').
https://github.com/django/django/blob/6bf85ff7e3b837378589e449ba27be8971d9b14c/django/forms/formsets.py#L263-L265

Reply:
For regular formsets, the docs already contain examples of passing formsets data with the prefix of `form` and show how to use prefixes in [this section](https://docs.djangoproject.com/en/1.11/topics/forms/formsets/#using-more-than-one-formset-in-a-view).

Do you think they need a separate example explicitly stating that the default prefix is `form`? If so I think either just after or just before [the management form](https://docs.djangoproject.com/en/1.11/topics/forms/formsets/#understanding-the-managementform) explanation would be a good place.

I added this example because InlineFormsets change the default formset prefix in a way that wasn't documented before.

Reply:
Yea, I guess it's not really clear where that "form-" bit comes from in the formset docs. If that were more clear, then I think we could omit the new examples here and point to the formset docs for examples explaining to substitute the prefix for "form-".

Reply:
I moved the example to the modelformsets docs, and just put a reference there in the InlineFormsets docs. Hopefully this one will be more clear.